db/model/Gdoc/htmlToEnriched.ts

```diff
@@ -596,6 +614,92 @@ function finishWpComponent(
                 }
             } else return { ...content, errors }
         })
+        .with("owid/summary", () => {
+            // Summaries can either be lists of anchor links, or paragraphs of text
+            // If it's a paragraph of text, we want to turn it into a callout block
+            // If it's a list of anchor links, we want to turn it into a toc block
+            const contentIsAllText =
+                content.content.find(
+                    (block) => "type" in block && block.type !== "text"
+                ) === undefined
+
+            if (contentIsAllText) {
+                const callout: EnrichedBlockCallout = {
+                    type: "callout",
+                    title: "Summary",
+                    text: content.content as EnrichedBlockText[],
+                    parseErrors: [],
+                }
+                return { errors: [], content: [callout] }
+            }
+
+            const contentIsList =
+                content.content.length === 1 &&
+                "type" in content.content[0] &&
+                content.content[0].type === "list"
+            if (contentIsList) {
+                const listItems = get(content, ["content", 0])
```

Comment:
it seems like "items" is missing and so list-based summary blocks (e.g. https://ourworldindata.org/working-hours) return an empty block (yet visible and expandable).

```suggestion
                const listItems = get(content, ["content", 0, "items"])
```

Reply:
Oof! Sorry about that! You're right, I removed it for the callout refactor and then forgot to change it back.